src/geojson/geojson.ml

```diff
@@ -81,6 +81,10 @@ module Make (J : Geojson_intf.Json) = struct
       let position = Fun.id
       let v position = position
       let parse_coords coords = J.to_array (decode_or_err J.to_float) coords
+
+      let bbox arr =
+        try Some (Array.append arr arr) with e -> ( match e with _ -> None)
```

Comment:
Note this could be simplified to:

```suggestion
        try Some (Array.append arr arr) with _ -> None
```

(but given my comments about simply parsing the bbox information rather than generating the bound box, I don't think we'll actually need this).